config/defaults.conf

```diff
@@ -93,6 +93,9 @@ transfer_paradigm = "frozen" // How to use pretrained model parameters during ta
 load_target_train_checkpoint = none  // If not "none", load the specified model_state checkpoint
                                      // file when starting do_target_task_training.
                                      // Supports * wildcards.
+load_eval_checkpoint = none  // If not "none", load the specified model_state checkpoint
+                              // file when doing do_full_eval
+
```

Comment:
Should also support wildcards. If we wind up with the glob-then-assert-then-assert logic in more than two places, it might be worth it to make it a utility method.